### PR TITLE
Rename docker image registry from openproject/community to openproject/openproject

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.org/work_packages/50623/activity

### CODE REVIEW

**Comment by oliverguenther** [comment]
Failing spec is unrelated

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -107,11 +107,6 @@ jobs:
       - name: Prepare docker files
         run: |
           cp ./docker/prod/Dockerfile ./Dockerfile
-
```
Comment (machisuji) [inline .github/workflows/docker.yml]: Is the removal of this intended @oliverguenther ?

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (machisuji) [inline test.sh]: accidental commit?

File: docs/installation-and-operations/misc/migration-to-postgresql13/README.md [hunk 3]
```diff
@@ -231,7 +231,7 @@ docker run -d -p 8080:80 --name openproject -e SECRET_KEY_BASE=secret \
   -v /var/lib/openproject/pgdata:/var/openproject/pgdata \
   -v /var/lib/openproject/assets:/var/openproject/assets \
   [...]
-  openproject/community:13
+openproject/openproject:14
```
Comment (machisuji) [inline docs/installation-and-operations/misc/migration-to-postgresql13/README.md]: the two spaces in the beginning should be kept!

**Review by machisuji — CHANGES_REQUESTED** [review]
Looks good overall, but I have some minor comments.

**Pull request merged**